docs/docs/disc-images.md

```diff
@@ -111,3 +111,47 @@ You can do this with utilities such as `dd`, `extract-xiso` or `fallocate` on su
     ```
 
     You can then use the `game.iso` with xemu.
+
+## Converting Digital Xbox 360 backwards compatible titles
```

Comment:
Do you know if this process also works on newer Xbox consoles?

Reply:
It does not as it uses a different file structure. The tools only works on original Xbox and 360 files. It would be wonderful if you can dump from a Xbox one. Maybe I can look into to see if there's some trickery, but I don't think it can.